spec/kitchen/driver/vagrant_spec.rb

```diff
@@ -210,6 +210,16 @@ def run_command(_cmd, options = {})
       expect(driver[:box_check_update]).to eq(true)
     end
 
+    it "sets :box_download_ca_cert to nil by default" do
+      expect(driver[:box_download_ca_cert]).to eq(nil)
+    end
+
+    it "sets :box_download_ca_cert to a custom value" do
+      config[:box_download_ca_cert] = "cacert.pem"
+
+      expect(driver[:box_download_ca_cert]).to eq("cacert.pem")
```

Comment:
probably because the path does not get resolved with `kitchen_root` until the `render_template` method.